### DISCUSSION TITLE
M5stickC Plus2

### DISCUSSION THREAD

**Discussion by stefo8** [raomin/ESPAltherma#387] [Q&A]
Hi,
now only the new Plus2 version is available, can you confirm that it is configured the same as the Plus version? 
Thank you. 
Sorry, I'm just starting out

**Comment by stefo8** [comment]
![image](https://github.com/raomin/ESPAltherma/assets/120152804/ce7326cb-06a8-4849-b712-79512825092f)

**Comment by raomin** [comment]
Yes, looks good!

**Reply by stefo8** [reply]
Thank you very much!
one last question: I searched in the other previous Discussions but I didn't understand if there is a device with an Ethernet socket with which to replace the M5stickC without modifying the guide you wrote. 
Thanks again!

**Comment by AshvanR** [comment]
Hi, are you sure this also works for the Plus2 version?
I am trying to upload the firmware to the stick but it gets stuck when writing the monitor part.
Do you have any idea's what is going wrong?

**Comment by wilyoung** [comment]
I can confirm this code does NOT work with the M5StickC Plus2 but works fine with the M5StickC Plus. I have both units.

**Comment by raomin** [comment]
Can you give some logs so I could possibly guess what needs to be adjusted?

**Comment by wilyoung** [comment]
Hi,
    The image appears to load fine but the lcd screen on the unit remains
black and I am unable to ping the unit. When I try to put any other image
on the plus2 unit that also doesn’t work after this image. If I put the
default factory image back on the plus2 it’s fine.
   The exact same image works fine loaded onto my m5stickc plus.
  Not sure how to get logs off plus2 unit with your image on.
Regards,
Bill

**Comment by MarkLofthouse** [comment]
Same here - found another thread that says the code doesn’t work on the Plus 2, can we get a view if it’s in the roadmap before I return the plus 2? Thanks.

**Comment by wilyoung** [comment]
I took the easy route and just used an mstickc plus.
Regards,
Bill

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
